demo/api/src/products/generated/product.resolver.ts

```diff
@@ -199,14 +198,14 @@ export class ProductResolver {
 
     @Mutation(() => Product)
     @SubjectEntity(Product)
-    async updateProductVisibility(
+    async updateProductStatus(
```

Comment:
do we need this mutation at all? updateProduct doesn (now) support partial updates

Reply:
We should base this decision upon the best practice we want to encourage: Create specific mutations for specific use cases, or use the generic update mutation for everything. I'd say use the generic mutation, especially since we added support for partial updates

Reply:
ok; removed it.